server/services/notificationService.test.ts

```diff
@@ -0,0 +1,55 @@
+import NotificationService, { type EmailPersonalisation } from './notificationService'
+import config from '../config'
+
+const notifyApi = {
+  sendEmail: jest.fn(),
+}
+
+describe('Notification service', () => {
+  let notificationService: NotificationService
+
+  beforeEach(() => {
+    notificationService = new NotificationService(notifyApi)
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  describe('Send feedback email', () => {
+    const feedbackEmail: EmailPersonalisation = {
+      username: 'A_USER',
+      prison: 'MDI',
+      feedback: 'Some feedback',
+    }
+
+    it('Details are retrieved for user', async () => {
```

Comment:
Think we can remove this test? Or add some assertions? For example, that the email is sent when optional fields (email) aren't passed? 😃

Reply:
Ah, realised in the service layer the email isn't actually optional as a default value is assigned and passed down from the controller is no email address is provided by user - have made amendments for this in the lates commit and removed this no unnecessary test